packages/ui-kit/src/components/SimpleFilter/SimpleFilter.stories.tsx

```diff
@@ -206,6 +181,19 @@ export const FreeSoloStory: Story = {
       freeSolo: true
     }
   },
+  decorators: [
+    (Story) => {
+      return (
+        <div>
+          <p style={{ color: 'var(--propel-text-primary)' }}>
+            <strong>Note:</strong> You can enable <code>freeSolo</code> in order to set values that are not in the
```

Comment:
I think I missed us introducing this terminology and prop. Should we not always enable "freeSolo"? Do we really need a prop?

Reply:
I think probably for most cases you'll want to use `freeSolo` mode, however I think maybe we should still give the possibility to disable it, wdyt if we make `freeSolo` the default but we still leave the prop so it's possible to disable it?